<?xml version="1.0"?>
<launch>
  <!-- <arg name="chain_start" default="base" />
  <arg name="chain_end" default="link_roll" />

  <param name="robot_description" command="cat $(find rviz_animator)/models/robocam.xml" /> -->

  <node name="ik_solver" pkg="ik_solver" type="ik_solver" output="screen"/>

</launch>
